src/App.jsx

```diff
@@ -9,32 +11,52 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [value, setValue] = useState('tab-1');
+  const [titleH1, setTitleH1] = useState('Tab 1');
+  const [text, setText] = useState('Some text 1');
```

Comment:
```suggestion
```
redundant states.
just manage selectedTabId and find selectedTab by id.
```
const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```
another way keep in state full tab instead only tab id.